schemas/app/inventory/StockMovementItem.json

```diff
@@ -26,6 +26,13 @@
       "target": "Location",
       "create": true
     },
+    {
+      "fieldname": "batchNumber",
```

Comment:
Why is there only one `batchNumber` field here?

If batches are enabled then the user could transfer from one batch to another batch.